src/commands/liquidity.ts

```diff
@@ -1,10 +1,11 @@
 import { SlashCommand } from 'slash-create';
 import fetch from 'node-fetch';
 import { formatLargeNumber } from '../utils';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
We actually should probably not use require at all. I expect you should be able to do something like this:

import { default as Discord } from 'discord.js';

Or maybe it's * as?

I'm fairly certain we don't need the 'require' here.

Reply:
I tried to convert it to an import but it breaks stuff.
```shell
PS D:\Javascript\Discord\million-discord-bot> npm run build

> million-token-bot@0.0.1 build
> tsc -p tsconfig.release.json

src/commands/liquidity.ts:115:19 - error TS2345: Argument of type '[{ name: string; value: string; Inline: boolean; }, { name: string; value: strin
g; Inline: boolean; }, { name: string; value: string; Inline: boolean; }, { name: string; value: string; Inline: boolean; }]' is not assignable to
parameter of type 'EmbedFieldData[] | EmbedFieldData[][]'.
  Type '[{ name: string; value: string; Inline: boolean; }, { name: string; value: string; Inline: boolean; }, { name: string; value: string; Inlin
e: boolean; }, { name: string; value: string; Inline: boolean; }]' is not assignable to type 'EmbedFieldData[][]'.
    Type '{ name: string; value: string; Inline: boolean; } | { name: string; value: string; Inline: boolean; } | { name: string; value: string; In
line: boolean; } | { name: string; value: string; Inline: boolean; }' is not assignable to type 'EmbedFieldData[]'.
      Type '{ name: string; value: string; Inline: boolean; }' is not assignable to type 'EmbedFieldData[]'.
        Object literal may only specify known properties, and 'name' does not exist in type 'EmbedFieldData[]'.

115                 { name: `Today's Volume:`, value: `${formatLargeNumber(volumeUSD_value)}`, Inline: true },
                      ~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error.
````